website/docusaurus.config.js

```diff
@@ -1,10 +1,11 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-const { resolve, dirname } = require("path");
+import { resolve, dirname } from "node:path";
```

Comment:
Waarom werkt dit 🤔 

de package `/website` is geen module (in package.json staat geen `"type": "module"), deze file is geen `.mjs` extensie. Dus hoe komt dat we hier kunnen importeren ipv requiren?

Reply:
Goed vraag, maar ik weet het niet. De aanpassingen in deze file waren een gevolg van mijn stoeien met `@typescript-eslint/no-require-imports`. Uiteindelijk heb ik die regel uitgezet, maar over het hoofd gezien dat ik de wijzigingen in `docusaurus.config.js` had gemaakt. Uiteindelijk heb ik `docusurus.config.js` weer teruggedraaid en de volgende configuratie in `eslint.config.mjs` gezet:

```javascript
"@typescript-eslint/no-require-imports": [
        "error",
        {
          allow: [
            "concurrently",
            "@angular-devkit/build-angular/plugins/karma",
            "karma-coverage",
            "karma-jasmine-html-reporter",
            "karma-chrome-launcher",
            "karma-jasmine",
            "path",
            "prism-react-renderer",
          ],
        },
      ],
``` 
De paden/namen in option `allow` zijn de `require`s waarover `@typescript-eslint/no-require-imports` viel in 

- packages/core/scripts/watch.js (concurrently)
- website/docusaurus.config.js (path, prism-react-renderer)
- angular-workspace/projects/component-library/karma.conf.js (de overige)